__main__.py

```
@@ -0,0 +1,600 @@
+        if b64_matches:
+            print("\n[*] Found potential base64 strings:")
+            for b64 in b64_matches[:5]:  # Show first 5
+                try:
+                    import base64
+                    decoded = base64.b64decode(b64).decode('utf-8')
+                    if 'flag' in decoded.lower() or 'ctf' in decoded.lower():
+                        print(f"  📦 {b64[:20]}... -> {decoded}")
+                except:
+                    pass
+
+def save_ctf_report(result: dict, filename: str):
```

Ruff (0.12.2)

Missing return type annotation for public function `save_ctf_report`

Add return type annotation: `None`

(ANN201)